test-definitions/fern/exhaustive/definition/types/object.yml

```diff
@@ -0,0 +1,46 @@
+types: 
+  ObjectWithOptionalField: #generic object that supports any type, makes it easier to use when testing 
+    properties: 
+      string: optional<string>
+      # integer: optional<integer>
+      # long: optional<long>
+      # double: optional<double>
+      # bool: optional<boolean>
+      # datetime: optional<datetime> 
+      # date: optional<date>
+      # uuid: optional<uuid>
+      # base64: optional<base64>
+      # list: optional<list>
+      # set: optional<set<string>>
+      # map: optional<map<integer, string>>
```

Comment:
why are all these commented out?

Reply:
was gonna ask u - wasn't sure we needed all the optional types for a simple object with optional fields test. I'll uncomment

Reply:
how else would be test that a list<integer> or list<string> are properly serialized inside an object?

Reply:
good point, uncommented!